roles/core/conman/molecule/default/molecule.yml

```diff
@@ -5,9 +5,29 @@ driver:
   name: docker
 platforms:
   - name: instance
+    groups:
+      - iceberg1
     image: docker.io/pycontribs/centos:7
     pre_build_image: true
 provisioner:
   name: ansible
+  inventory:
+    host_vars:
+      instance:
+        j2_current_iceberg: iceberg1
+        equipment_profile:
+          equipment_type: server
+          authentication:
+            equipment_user: 'ADMIN'
+            equipment_password: 'ADMIN'
+          operating_system:
```

Comment:
Should not be necessary, we can get similar information with `ansible_distribution*` facts.

Reply:
Agreed, this role can use facts.